apps/src/StudioApp.js

```diff
@@ -63,10 +63,10 @@ var StudioApp = function () {
   // The following properties get their non-default values set by the application.
 
   /**
-  * Whether to alert user to empty blocks, short-circuiting all other tests.
-  */
-  // TODO (br-pair) : this isnt actually a constant
-  this.CHECK_FOR_EMPTY_BLOCKS = undefined;
+   * Whether to alert user to empty blocks, short-circuiting all other tests.
+   * @member {boolean}
+   */
+  this.checkForEmptyBlocks_ = false;
```

Comment:
It seemed appropriate to initialize this to false; it's a boolean value, never actually checked for existence or against "undefined", and it was a falsy value before.